doc-rosindex.yaml

```diff
@@ -0,0 +1,16 @@
+%YAML 1.1
+# ROS buildfarm doc-build file
+---
+documentation_type: docker_build
+doc_repositories:
+- https://github.com/ros2/rosindex.git
```

Comment:
```suggestion
- https://github.com/ros-infrastructure/rosindex.git
```

Reply:
The repo hadn't been moved when I dropped the link. :+1: